tests/test_routes.py

```diff
@@ -79,6 +80,7 @@ def test_search(client):
         assert len(response_json) == len(rows)
 
 
+@pytest.mark.skip(reason="used for tdd only")
```

Comment:
Should skip the test from now on. Since the test is only testing config, it'll probably break a lot. Was originally used for TDD, so we don't really need it unless someone wants to use it for development.